### DISCUSSION TITLE
ImageMagick Adobe InDesign Project file support

### DISCUSSION THREAD

**Discussion by lutonian** [ImageMagick/ImageMagick#6236] [General]
Does anyone know whether ImageMagick supports Adobe InDesign Project files on Windows? I attempted to get it working today, however, it reported that the following DLL file was missing:-

IM_MOD_RL_INDD_.dll

**Comment by snibgo** [comment]
For a list of formats that IM supports, see https://www.imagemagick.org/script/formats.php . I don't see InDesign listed there.

**Reply by lutonian** [reply]
Thanks for the info Snibgo, much appreciated 👍
